extensions/data-workspace/package.nls.json

```diff
@@ -10,5 +10,6 @@
 	"projects-view-no-workspace-content": "No workspace open, create new or open existing to get started.\n[Create new](command:projects.new)\n[Open existing](command:projects.openExisting)\nTo learn more about SQL Database projects [read our docs](https://aka.ms/azuredatastudio-sqlprojects).",
 	"projects-view-no-project-content": "No projects found in current workspace.\n[Create new](command:projects.new)\n[Open existing](command:projects.openExisting)\nTo learn more about SQL Database projects [read our docs](https://aka.ms/azuredatastudio-sqlprojects).\n",
 	"open-existing-command": "Open existing",
-	"projects.defaultProjectSaveLocation": "Full path to folder where new projects are saved by default."
+	"projects.defaultProjectSaveLocation": "Full path to folder where new projects are saved by default.",
+	"projects.showNotAddedProjectsInWorkspacePrompt": "Always show information message when the current workspace folders contain a project that have not been added to the workspace's projects."
```

Comment:
... when the current workspace folders contain **projects** that have... instead of **a project**?